.github/workflows/release.yml

```diff
@@ -0,0 +1,104 @@
+---
+name: "Release"
+on: # yamllint disable-line rule:truthy rule:comments
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    name: "Build package with poetry"
+    runs-on: "ubuntu-24.04"
```

Comment:
`ci.yml` is using `ubuntu-latest`. Can we standardize on that across the board (in the app cookie too)